### PR TITLE
Resolving issue #045

### PR DESCRIPTION
**Pull request by thomasstorm**

### CODE REVIEW

File: xcube_geodb/core/geodb.py [hunk 1]
```diff
@@ -969,58 +1007,73 @@ def list_my_grants(self) -> Union[DataFrame, Message]:
...
-            >>> geodb.add_property(collection='[MyCollection]', properties=properties)
+            >>> geodb.add_property(collection='[MyCollection]', \
+                                   properties=properties)
         """
 
         self._refresh_capabilities()
 
         database = database or self.database
-        collection = database + '_' + collection
+        dn = database + '_' + collection
```
Comment (AliceBalfanz) [inline xcube_geodb/core/geodb.py:1059]: what is dn? I would prefer a more self describing variable name.

Comment (thomasstorm) [reply xcube_geodb/core/geodb.py:1064]: dn = distinguished name (I guess)

Yes, I'd also prefer a better name, but I put it here for consistency reasons over the geodb client (it's named like this everywhere). Renaming is out of the scope of this PR.

**Review by AliceBalfanz — APPROVED** [review]
Thank you so much for all the adjustments! I think this will add a lot of value for the xcube geoDB users!

**Review by forman — APPROVED** [review]
Looks all good to me :)
Please add a PR description next time that includes a summary of the changes.

**Pull request merged**